config/hardware/extruder/orbiter2.0.cfg

```diff
@@ -0,0 +1,27 @@
+[extruder]
+# Orbiter 2.0 Gear Ratio
+# new_rd = previous_rd * mesured_distance / requested_distance
+rotation_distance: 4.637
+gear_ratio: 7.5:1
+microsteps: 16
+full_steps_per_rotation: 200
+
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+max_extrude_only_distance: 110
```

Comment:
can you add also the following line (needed for the new purge line system):

`max_extrude_cross_section: 5`